README.md

```diff
@@ -1 +1,106 @@
-# Nextcloud-OpenProject-App repository
\ No newline at end of file
+# OpenProject as Nextcloud's External App
+
+## Manual Installation
+For the manual installation of `OpenProject` as an external application of `Nextcloud`, make sure that your `Nextcloud` as well as `OpenProject` instance is up and running.
```

Comment:
I think we need to mention that openproject and nextcloud are need to set up locally or through docker

Reply:
No this is the manual installation part. for docker there will be another readme which is the next step